### DISCUSSION TITLE
Update libGDX version in VTM

### DISCUSSION THREAD

**Discussion by wolfgang-ch** [mapsforge/vtm#971] [Ideas]
That gltf 2 models can be loaded in libGDX with [gdx-gltf](https://github.com/mgsx-dev/gdx-gltf), vtm would need the gdx version [libGDX 1.9.11+](https://github.com/mgsx-dev/gdx-gltf#user-content-compatibility), currently [version 1.9.10](https://github.com/mapsforge/vtm/blob/332a694d653db2d49e044f3b91945e014766319a/build.gradle#L9) is used.

Are there plans to use libGDX 1.9.11+ in vtm?

**Comment by devemux86** [comment]
libGDX appears to be at version `1.11.0`.

Last time checked it is more than a version change, additional VTM changes are needed.
Can you check them and post a pull request with all VTM modules building successfully?

Also, there are reports that VTM no longer runs on macOS.
Maybe because of libGDX or VTM macOS native libs.

**Reply by wolfgang-ch** [reply]
I'll try to use libGDX `1.11.0` but I can test it only on Windows and Linux (VM)

**Reply by wolfgang-ch** [reply]
I've created this PR https://github.com/mapsforge/vtm/pull/972

**Comment by wolfgang-ch** [comment]
With [gdx-gltf](https://github.com/mgsx-dev/gdx-gltf), and libGDX `1.11.0`, the VTM map can display animated models

https://user-images.githubusercontent.com/1283445/204142511-ec7aabda-ea9e-4c7e-9b27-bf0db7f9ed08.mp4

https://github.com/mytourbook/mytourbook/issues/516#issuecomment-1328267295
